src/internal/connector/onedrive/collection.go

```diff
@@ -197,7 +197,8 @@ func (oc Collection) DoNotMergeItems() bool {
 type UserPermission struct {
 	ID         string     `json:"id,omitempty"`
 	Roles      []string   `json:"role,omitempty"`
-	Email      string     `json:"email,omitempty"`
+	Email      string     `json:"email,omitempty"` // DEPRECATED: Replaced with UserID in newer backups
```

Comment:
nice, I like the comment and good to know we won't see JSON deserialization errors due to not having the field